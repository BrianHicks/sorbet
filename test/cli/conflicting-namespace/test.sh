#!/bin/bash
set -eu

set +e
main/sorbet --silence-dev-message --stripe-mode --stripe-mode-namespace-collision-check-experimental \
  test/cli/conflicting-namespace/{a,b}.rb 2>&1
set -e

echo "without --stripe-mode-namespace-collision-check-experimental"
main/sorbet --silence-dev-message --stripe-mode \
  test/cli/conflicting-namespace/{a,b}.rb 2>&1
