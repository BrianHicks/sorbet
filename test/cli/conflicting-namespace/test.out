test/cli/conflicting-namespace/b.rb:4: `Foo::Bar` is a common prefix in multiple files https://srb.help/4022
     4 |  module Bar
          ^^^^^^^^^^
  Note:
    Additional file: `test/cli/conflicting-namespace/a.rb`
Errors: 1
without --stripe-mode-namespace-collision-check-experimental
No errors! Great job.
